### ISSUE TITLE
Add asctime word to logging formatter

### ISSUE THREAD

**Issue by vikingUnet** [dalibo/ldap2pg#428]
Hello!
Sometimes it is usefull to see datetime of starting each launch and each operation in logfile.
Can we add this optional ability to logging iso timestamp for each operation? I have an idea how in can be implemented :D

**Comment by bersace** [comment]
Hi @vikingUnet , thanks for reaching. `DEBUG=y` is a hidden feature enabling timestamp on log message.

**Comment by vikingUnet** [comment]
DEBUG feature like verbose option produce wide log statistics, but there is no datetime field in it =(

**Comment by bersace** [comment]
It looks like I missed the point. You want a startup message stating the datetime of ldap2pg launch ?

something like

``` console
$ ldap2pg
Starting ldap2pg v5.9 at 2022-10-27 14:16:89 EST.
...
```

**Comment by vikingUnet** [comment]
It will be nice to at least print datetime in startup message.
But i suggest to starting each line to stdout with timestamp, like it will be if log formatter template is
`'format': '[%(asctime)s] %(message)s'`
[link to line in source code to that place](https://github.com/dalibo/ldap2pg/blob/fc0c069fe53ac54c77e470dcbfd35a93ba5def46/ldap2pg/config.py#L639)

**Comment by bersace** [comment]
I'm fine with it.

**Comment by bersace** [comment]
Would you mind the propose a patch ? I suggest something more syslog/postgresql-like :

`%(asctime)s $(levelname)s:  %(messages)`

What do you think of this ?

**Comment by vikingUnet** [comment]
Yes, i would like to produce patch, it will be nice! Syslog style is ok.
In your suggestion "%(asctime)s" template will be working only in debug mode?
Or it will be printing in both modes?

**Comment by vikingUnet** [comment]
Add [PR](https://github.com/dalibo/ldap2pg/pull/429) for patch it. Can you check it, please?

**Comment by vikingUnet** [comment]
Add commits to resolve comments in PR.

**Comment by bersace** [comment]
Fixed by #429
